src/eventer/eventer_SSL_fd_opset.c

```diff
@@ -1197,7 +1197,7 @@ eventer_SSL_close(int fd, int *mask, void *closure) {
   } else {
     rv = close(fd);
   }
-  if(mask) *mask = 0;
+  *mask = 0;
```

Comment:
Do we know for a fact that `mask` isn't NULL here? If no, seems like this could introduce a crash bug.

Reply:
We use it up above as if it is not-NULL, so yes.